docs/developers/updates.md

```diff
@@ -26,6 +24,67 @@ function removeData(chart) {
 }
 ```
 
+## Opdating Options
```

Comment:
Opdating -> Updating

Reply:
Thanks, I really should install a spell checker for my editor : P